src/views/MorphicBarEditor.vue

```diff
@@ -480,11 +476,11 @@ export default {
       }
     },
     getMakeAButtons: function () {
-      let buttons = []
+      const buttons = []
```

Comment:
Same here, empty constant that you are starting modifying right away, makes no sense.